integration_tests/klog_test.go

```diff
@@ -214,13 +225,19 @@ func TestDestinationsWithDifferentFlags(t *testing.T) {
 				}
 
 				// check files in log_dir
-				for level, file := range logFileName {
-					logfile := filepath.Join(logdir, file) // /some/tmp/dir/main.WARNING
+				for level, levelName := range logFileLevels {
+					binaryName := "main" + binaryFileExtention
```

Comment:
we can move the `binaryFileExtention` variable definition in `TestDestinationsWithDifferentFlags` for now.